lizrd/train/scheduler.py

```diff
@@ -10,15 +10,19 @@ def get_scheduler(
 ) -> "AbstractLRScheduler":
     if ratios_in_group_order is None:
         ratios_in_group_order = [1.0]
+    if args.lr_warmup_percent is not None:
+        assert not args.lr_warmup_steps  # cannot set both
```

Comment:
You should give a message instead of a comment

Reply:
ok, gonna give an exception

Reply:
I meant somthing like this
```suggestion
        assert not args.lr_warmup_steps, "You cannot set both lr_warmap_percent and lr_warmap_steps"
```
alternatively such assert could be made using argparse lib